frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/utilization-card/utilization-card.tsx

```diff
@@ -130,7 +130,7 @@ const UtilizationCard: React.FC<DashboardItemProps> = ({
           <UtilizationItem
             title="Used Capacity"
             data={capacityStats}
-            humanizeValue={humanizeBinaryBytesWithoutB}
+            humanizeValue={humanizeBinaryBytes}
```

Comment:
Please add the binaryBytesProp here. Somehow I missed it here.

Reply:
Okay, added.

Reply:
You forgot to import it. I guess I can take that up in a different PR.